kubeflow/openvino/openvino.libsonnet

```diff
@@ -0,0 +1,101 @@
+{
+  local k = import "k.libsonnet",
+  local util = import "kubeflow/core/util.libsonnet",
+  local service = k.core.v1.service,
+  local deployment = k.apps.v1beta1.deployment,
+  local container = deployment.mixin.spec.template.spec.containersType,
+
+  new(_env, _params):: {
+    local params = _env + _params {
+      labels: {
+        role: _params.name,
+      },
+      imageURL: _params.registry + "/" + _params.repoPath + "/" + _params.image,
+    },
+    params:: params,
+
+    local ovService =
+      service.new(
+        name=params.name,
+        selector=params.labels,
+        ports=service.mixin.spec.portsType.newNamed("ov", params.servicePort, params.targetPort),
+      ).withType(params.serviceType) +
+      service.mixin.metadata.
+        withNamespace(params.namespace).
+        withAnnotationsMixin({
+        "getambassador.io/config":
+          std.join("\n", [
+            "---",
+            "apiVersion: ambassador/v0",
+            "kind:  Mapping",
+            "name: ov-mapping-" + params.name + "-get",
+            "prefix: / " + params.name + "/",
```

Comment:
should be prefix has something before name?
like openvino, or ov

Reply:
it will end up as /openvino/ - so similar to jupyter's /hub/ or /user/

Reply:
For hub, it has /hub/ in the prefix.
I think we want /openvino/ in the path, so the prefix should be something like
`"/openvino/" + params.name + "/"` , right?

Reply:
i agree - but params.name is "openvino" ... where you thinking /openvino/openvino/ ?

Reply:
Why would params.name be openvino?
Shouldn't that be more model specific, like myModelA?

Or you will only deploy one openvino deployment in the cluster?
Even in that case, I still think we should have /openvino/ because params.name is too arbitrary, and user can choose some name that will collide with other path.
WDYT?

Reply:
I believe there is 1 openvino deployment in the cluster at this time